document/core/exec/runtime.rst

```diff
@@ -542,21 +542,20 @@ to their associated branch *targets*, each of which is expressed syntactically a
 :ref:`instructions <syntax-instr>` possibly following a :ref:`tag address <syntax-tagaddr>`.
 If there is no :ref:`tag address <syntax-tagaddr>`, the instructions of that handler clause correspond to a |CATCHALL| clause.
 
-.. todo::
-   Add prose for delegating handlers.
+Delegating handlers start with the identifier |DELEGATEadm| and carry a label index,
+indicating the outer block in which any caught exceptions will be thrown (delegated to).
```

Comment:
See my email suggestion regarding a slight refactor of this syntax.

Reply:
I wanted to change this prose here to match #226, but on second thought it doesn't make sense without the rest of the changed text. So I am reverting these changes and adding this missing prose to #226 instead.

Reply:
Done.